robotiq_c_model_control/nodes/autodetect_gripper_node.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+"""
+Copyright 2018 by READY Robotics Corporation.
+All rights reserved. No person may copy, distribute, publicly display, create derivative works from or otherwise
+use or modify this software without first obtaining a license from the READY Robotics Corporation.
+"""
```

Comment:
Here is another where the copyright is a string instead of a comment.

Reply:
Thank you! I clearly missed the nodes directory!